### ISSUE TITLE
Table filters do not update user on state change

### ISSUE THREAD

**Issue by sryeluri** [telerik/kendo-angular#2043]
OS: Windows 1809

Browser: Chrome 70.0.3538.77
Assistive Technology: NVDA

Steps:
1. Navigate to Application
2. Press tab until focus is given to the table.
3. Navigate to any filter 
4. Enable different filter options by pressing 'Enter'
5. Observe screen reader feedback.

Actual Results: 
Although current state is supplied upon landing on filter (Good) No screen reader feedback is provided up changing the filter state.

Expected Results:
Changes to filter state should be communicated via screen reader, such as - 'Ascending' 'Descending' and 'Selected'

**Comment by ipeshev** [comment]
@sryeluri thank you for the bug report.

**Comment by tsvetomir** [comment]
Merging with https://github.com/telerik/kendo-angular/issues/1710

**Comment by kraay** [comment]
@tsvetomir Created bug [#4861](https://github.com/telerik/kendo-ui-core/issues/4861) in kendo-ui-core as our application uses AngularJS.
